### ISSUE TITLE
Add content description for each image region in ImageRegionSelectionInteraction

### ISSUE THREAD

**Issue by FareesHussain** [oppia/oppia-android#3708]
**Description**
`ImageRegionSelectionInteraction` does not contain `contentDescription` and therefore fails on accessibility scanner

<img src="https://github.com/oppia/oppia-android/assets/9396084/53569267-be0d-48ad-bf01-c12704fe7a27" width="250" />

**Steps to reproduce**
1. Open `First Test Topic`
2. Open `Image Region Selection Exploration`
3. Using [accessibility scanner](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide#using-a11y-scanner-in-android) take screenshot of the screen which shows all planets.
4. Notice that the scanner fails with various errors

**Expected solution**
The scanner should should 0 errors.

For each image region we should add content description. Follow files can help you get started:
- app/src/main/java/org/oppia/android/app/player/state/ImageRegionSelectionInteractionView.kt
- app/src/main/res/layout/image_region_selection_interaction_item.xml

**Comment by BenHenning** [comment]
I'm taking this to mean fixing the content descriptions in the test exploration that includes an image region selection instance.
